tests/test_apiclient.py

```diff
@@ -504,6 +504,20 @@ def test_find_users_by_supplier_id(self, data_client, rmock):
 
         assert user == self.user()
 
+    def test_find_users_not_possible_with_supplier_id_and_role(self, data_client, rmock):
+        with pytest.raises(ValueError):
+            data_client.find_users(supplier_id=123, role='buyer')
+
+    def test_find_users_by_role(self, data_client, rmock):
+        rmock.get(
+            "http://baseurl/users?role=buyer",
+            json=self.user(),
+            status_code=200)
+
+        user = data_client.find_users(role='buyer')
+
```

Comment:
All the tests for finding or getting users seems to be doing it this way? Example [here](https://github.com/alphagov/digitalmarketplace-apiclient/pull/33/files#diff-6ec10ad3428f70a1a2ae819921ff6b45R537).